docs/content/overview/cdc-pipeline.md

```diff
@@ -0,0 +1,107 @@
+
+Since events flow from the upstream to the downstream in a pipeline manner, the data synchronization task is also referred to as a **Data Pipeline**. A Data Pipeline consists of a Data Source, Route, Transform and Data Sink. The transform can modify events to some extent, and the router can remap the table IDs corresponding to events.
+
+Next, let's provide a detailed introduction to the concepts you need to know when using the CDC pipeline.
+
+### Table ID
+When connecting to external systems, it is necessary to establish a mapping relationship with the storage objects of the external system. This is what Table Id refers to.
+
+To be compatible with most external systems, the Table ID is represented by a 3-tuple, (namespace, schemaName, table). Connectors need to establish the mapping between Table ID and storage objects in external systems. For instance, a database table in MySQL/Doris is mapped to (null, database, table), and a topic in a message queue system such as Kafka is mapped to (null, null, topic).
+
+### Data Source
+Data Source is used to access metadata and read the changed data from external systems.
+A Data Source can read data from multiple tables simultaneously.
+
+To describe a data source, the follows are required:
+* Type: The type of the source, such as MySQL, Postgres.
+* Name: The name of the source, which is user-defined (optional, with a default value provided).
+* Other custom configurations for the source.
+
+
+For examle, we can use yaml to define a mysql source:
+```yaml
+source:
+  type: mysql
+  name: mysql-source   #optional，description information
+  host: localhost
+  port: 3306
+  username: admin
+  password: pass
+  tables: adb.*, bdb.user_table_[0-9]+, [app|web]_order_.*
+```
+The ability to synchronize the entire library comes from the implementation of Flink source. After obtaining the user configuration, the Flink source connector needs to be able to send all data in a database downstream according to the specified data type.
+
+### Data Sink
+The Data Sink is used to modify metadata and write change data to external systems. A Data Sink can write to multiple tables simultaneously.
+
+To describe a data sink, the follows are required:
+* Type: The type of the sink, such as MySQL or PostgreSQL.
+* Name: The name of the sink, which is user-defined (optional, with a default value provided).
+* Other custom configurations for the sink.
+
+
+For examle, we can use yaml to define a kafka sink:
+```yaml
+sink:
+  type: kafka
+  name: mysink-queue           	# Optional parameter for description purpose
+  bootstrap-servers: localhost:9092
+  auto-create-table: true      	# Optional parameter for advanced functionalities
+```
+
+### Route
+Routing, for each change event, specifies the target table ID of the event. The most typical application scenario is the merge of sub-databases and sub-tables, routing multiple upstream source tables to the same result table.
```

Comment:
Route specifies the target table ID of each event. The most typical scenario is the merge of sub-databases and sub-tables, routing multiple upstream source tables to the same sink table.